Makefile

```diff
@@ -12,10 +12,13 @@ install: ## Install
 .PHONY: build
 build: ## Build the project
 	@echo 'Building the project'
-	@brunch b --production
+	@node_modules/brunch/bin/brunch b --production
```

Comment:
Why not use `@node_modules/.bin/brunch b --production`?

Reply:
Because I did not know that `node_modules/.bin` contains all binaries